-VECTOR ERASE ITER TEST-

In file included from ./tmp/outfile_vec_11.cpp:1:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/iostream:37:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ios:215:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__locale:14:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/string:504:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/string_view:175:
In file included from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__string:57:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/algorithm:1274:56: error: no type named 'value_type' in 'std::__1::iterator_traits<ft::random_access_iterator<int> >'
    typedef typename iterator_traits<_InputIterator2>::value_type __v2;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~
./tmp/outfile_vec_11.cpp:48:13: note: in instantiation of function template specialization 'std::__1::equal<std::__1::__wrap_iter<int *>, ft::random_access_iterator<int> >' requested here
                if (!std::equal(v1.begin(),v1.end(),v2.begin()))
                          ^
1 error generated.
